fmriprep/workflows/bold/resampling.py

```diff
@@ -172,47 +180,43 @@ def select_target(subject_id, space):
         mem_gb=DEFAULT_MEMORY_MIN_GB,
     )
 
-    joinnode = pe.JoinNode(
-        niu.IdentityInterface(fields=["surfaces", "target"]),
-        joinsource="itersource",
-        name="joinnode",
-    )
-
-    outputnode = pe.Node(
-        niu.IdentityInterface(fields=["surfaces", "target"]),
-        name="outputnode",
+    ds_bold_surfs = pe.MapNode(
+        DerivativesDataSink(
+            base_directory=output_dir,
+            extension=".func.gii",
+            TaskName=metadata.get('TaskName'),
+            **timing_parameters,
+        ),
+        iterfield=['in_file', 'hemi'],
+        name='ds_bold_surfs',
+        run_without_submitting=True,
+        mem_gb=DEFAULT_MEMORY_MIN_GB,
     )
+    ds_bold_surfs.inputs.hemi = ["L", "R"]
 
-    # fmt: off
     workflow.connect([
         (inputnode, get_fsnative, [
             ("subject_id", "subject_id"),
             ("subjects_dir", "subjects_dir")
         ]),
         (inputnode, targets, [("subject_id", "subject_id")]),
         (inputnode, itk2lta, [
-            ("source_file", "reference"),
+            ("bold_t1w", "moving"),
             ("fsnative2t1w_xfm", "in_xfms"),
         ]),
-        (get_fsnative, itk2lta, [("T1", "moving")]),
+        (get_fsnative, itk2lta, [("T1", "reference")]),
```

Comment:
Note that this reversal fixes a bug introduced in 66cb0bb34a3233e6bf2827a43b01ea8b8039f889, which is only part of `:next`. The bug was not discovered because this workflow was not hooked up before now.